### PR TITLE
added systemd service file

### PR DESCRIPTION
**Pull request by DirectorX**
service file for systemd systems #55

### CODE REVIEW

File: initfiles/systemd/gitautodeploy.service [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[Unit]
+Description=GitAutoDeploy
+
+[Service]
+User=root
```
Comment (alexandernst) [inline initfiles/systemd/gitautodeploy.service]: Shouldn't be run as root.

**Comment by DirectorX** [comment]
are you sure?
```
DAEMON_UID=root
DAEMON_GID=root
```
from sys-v init file

File: initfiles/systemd/gitautodeploy.service (hunk 1 above)
Comment (DirectorX) [inline initfiles/systemd/gitautodeploy.service]: are you sure?
```
DAEMON_UID=root
DAEMON_GID=root
```
from sys-v init file

Comment (alexandernst) [inline initfiles/systemd/gitautodeploy.service]: That is also wrong, sadly. This process shouldn't never run as root, but as it's own user/group.

**Comment by alexandernst** [comment]
Can you add a few lines in the `README.md` about how to set this up, plelase?

File: initfiles/systemd/gitautodeploy.service (hunk 1 above)
Comment (DirectorX) [inline initfiles/systemd/gitautodeploy.service]: specific user or `www-data` ?

Comment (alexandernst) [inline initfiles/systemd/gitautodeploy.service]: Hard to say... I guess `www-data` should be fine, as long as it exists, so make sure to add that as a node in the `README.md` too!

**Comment by DirectorX** [comment]
@alexandernst ping

**Comment by alexandernst** [comment]
Oops, sorry, forgot about this completely. Everything looks good, merging!

**Pull request merged**